css-brochure-form/styles.css

```diff
@@ -0,0 +1,72 @@
+body {
+  font-family: sans-serif, "Brandon Grotesque";
+  font-style: normal;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+form {
+  width: 28%;
+  margin: auto;
+}
+
+input,
+select {
+  margin: 6px;
+  border-radius: 5px;
+  width: 90%;
+  border-width: 50%;
+  margin-right: 50%;
+  padding: 5px;
+  align-content: center;
+  font-weight: lighter;
+}
+
+label {
+  font-weight: 400;
+  margin: 10px;
+  font-size: x-small;
+  margin-top: 75%;
```

Comment:
😱 😱 😱 😱 

That's a **_lot_** of margin. Why is this necessary? There is likely a better way to accomplish this.